en/identity-server/next/mkdocs.yml

```diff
@@ -967,7 +969,9 @@ nav:
       - Identity recovery API: apis/organization-apis/org-account-recovery.md
       - Idle accounts identification API: apis/organization-apis/org-idle-accounts-identification-rest-api.md
       - Invite parent organization's users API: apis/organization-apis/organization-user-invitation.md
-      - Notification sender API: apis/organization-apis/notification-sender-rest-api.md      
+      - Notification sender API:
```

Comment:
Shall we setup redirections for the removed path?

Reply:
The path was not removed it is listed here https://github.com/wso2/docs-is/pull/5320/files#diff-53653710e1f22073d92c59884b21140096c8d2edee68f8d23789ae33e82f1d2dR973, as now we have two versions